.github/workflows/windows-ci.yml

```diff
@@ -28,4 +28,4 @@ jobs:
       run: dotnet build --configuration Release --no-restore
 
     - name: Test ${{ matrix.version }}
-      run: dotnet test **\bin\**\${{ matrix.version }}\*Tests.dll --configuration Release --no-build --logger:"console;verbosity=detailed"
+      run: dotnet test **\bin\**\${{ matrix.version }}\*Tests.dll --configuration Release --no-build --logger:"console;verbosity=detailed"  --filter "Platform=Any|Platform=Windows"
```

Comment:
for now, we can proceed.
 But as a general rule, the common workflows should not be required to be modified to meet the needs of individual components. 
Consider using an approach similar to the below, to detect OS and skip the tests.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs#L40

Please open an issue to track this, and we can unblock this PR.

Reply:
Okay! https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/281